docker/docker-compose-deployment/docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+services:
+
+  coatrack-database:
+    image: postgres:${POSTGRES_VERSION}
+    container_name: ${DATABASE_CONTAINER_NAME}
+    volumes:
+      - ${DATABASE_VOLUME}:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+
+  coatrack-config-server:
+    image: coatrack/coatrack-config-server:${TAG}
+    container_name: ${CONFIG_SERVER_CONTAINER_NAME}
+    volumes:
+      - ${PROXY_CONFIG_FILES_VOLUME}:${PROXY_CONFIG_FILES_FOLDER}:ro
+    depends_on:
+      - ${DATABASE_CONTAINER_NAME}
+    ports:
+      - 8998:8998
+    entrypoint: ["sh", "-c", "sleep 5 && java -Djava.security.egd=file:/dev/./urandom \
+      -jar ${JAR_DIR}/coatrack-config-server-${TAG}.jar \
+      --ygg.admin.config.access.user.name=${CONFIG_SERVER_USERNAME} \
+      --ygg.admin.config.access.user.password=${CONFIG_SERVER_PASSWORD} \
+      --spring.profiles.active=native \
+      --spring.cloud.config.server.native.searchLocations=file://${PROXY_CONFIG_FILES_FOLDER} \
+      --spring.datasource.username=postgres \
+      --spring.datasource.password=${POSTGRES_PASSWORD} \
+      --spring.datasource.url=jdbc:postgresql://${DATABASE_CONTAINER_NAME}/ygg_config_server \
+      --spring.datasource.driver=org.postgresql.Driver \
+      --spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect"]
+
+  coatrack-admin:
+    image: coatrack/coatrack-admin:${TAG}
+    container_name: ${ADMIN_CONTAINER_NAME}
+    volumes:
+      - ${PROXY_CONFIG_FILES_VOLUME}:${PROXY_CONFIG_FILES_FOLDER}
+    depends_on:
+      - ${DATABASE_CONTAINER_NAME}
+    ports:
+      - 8080:8080
+    entrypoint: [ "sh", "-c", "sleep 5 && java -Djava.security.egd=file:/dev/./urandom \
+      -jar ${JAR_DIR}/coatrack-admin-${TAG}.jar \
+      --spring.datasource.url=jdbc:postgresql://${DATABASE_CONTAINER_NAME}/coatrack \
+      --spring.datasource.username=postgres \
+      --spring.datasource.password=${POSTGRES_PASSWORD} \
+      --spring.jpa.hibernate.ddl-auto=update \
+      --spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect \
+      --spring.datasource.initialization-mode=always \
+      --ygg.admin.database.insertSampleDataOnStartup=${INSERT_SAMPLE_DATA_ON_STARTUP} \
+      --ygg.proxy.generate-bootstrap-properties.spring.cloud.config.uri=http://localhost:8998 \
+      --spring.cloud.config.uri=http://${CONFIG_SERVER_CONTAINER_NAME}:8998 \
+      --spring.cloud.config.username=${CONFIG_SERVER_USERNAME} \
+      --spring.cloud.config.password=${CONFIG_SERVER_PASSWORD} \
+      --ygg.proxy.executable-jar.template.url=file://${JAR_DIR}/coatrack-proxy- \
+      --proxy.config.files.folder=${PROXY_CONFIG_FILES_FOLDER} \
+      --is-run-via-docker-compose-setup=true"]
+
+  coatrack-example-gateway:
```

Comment:
This is actually used for the example-deployment tutorial which is not documented yet, because I removed that section from the README. We wanted to add it later, when this PR is merged and the deployment of your own CoatRack instance using dockerhub images works.